src/link/colorScheme.js

```diff
@@ -0,0 +1,77 @@
+import wikipediaPreview from 'wikipedia-preview';
+
+const isWPDarkModePluginActive = () => {
+    const htmlTag = document.documentElement;
+    // TODO also need to check for attribute as well as class 
+    // to correctly read on the first render if already in dark mode
+    return htmlTag.classList.contains( 'wp-dark-mode-active' );
+}
+
+export const getColorScheme = () => {
+	console.log('getColorScheme...');
+    if (
+		window.matchMedia( '(prefers-color-scheme: dark)' ).matches ||
```

Comment:
Why prefixed with `window.`? As far as I understand it's not needed and prevents linters from catching typos.

Reply:
Actually it seems we don't need to check `matchMedia` here at all, the `prefers-color-scheme` media query in the core library CSS takes care of that